README.md

```diff
@@ -3,15 +3,62 @@
 This repository creates the possibility of:
 - building cross-platform RonDB images
 - running local (non-production) RonDB clusters with docker-compose
-- benchmarking RonDB with Sysbench and DBT2
+- benchmarking RonDB with Sysbench and DBT2 on localhost
 - demo the usage of managed RonDB (upcoming)
 
 To learn more about RonDB, have a look at [rondb.com](https://rondb.com).
 
 ## Quickstart
 
 Dependencies:
-- Docker, docker-compose, Docker Buildx
+- Docker, docker-compose, Docker Buildx (if you use DockerHub this dependency isn't there)
+
+The run.sh script makes it very easy to start a RonDB cluster in Docker Compose using 3 commands:
+1. git clone https://github.com/logicalclocks/rondb-docker rondb-docker
+2. cd rondb-docker
+3. ./run.sh
+
+The run.sh makes it very easy to run with 5 different user profiles.
```

Comment:
Do mention that you mean 2.5 GB of memory **_allowed to the Docker engine_** (in case that is what you mean).

Reply:
<img width="1003" alt="Screenshot 2023-02-22 at 20 19 21" src="https://user-images.githubusercontent.com/31848129/220735974-4a38c81d-5831-434c-8af1-776d0c2cce79.png">